DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: pkgdown
 Title: Make Static HTML Documentation for a Package
-Version: 1.5.1.9000
+Version: 1.5.1.9001.1
```

Comment:
Can you please leave as is

Reply:
Sure, i had some silly internal debugging reason for the extra version level because I've been using my forks in my enterprise packages, and i forgot that i still had an open PR on it